drv/vsc85xx/src/vsc85x2.rs

```diff
@@ -122,11 +129,26 @@ impl<'a, P: PhyRw> Vsc85x2Phy<'a, P> {
         }
     }
 
-    pub fn mac_tx_rx_good(&mut self) -> Result<(Counter, Counter), VscError> {
-        // Configure the PHY to read MAC side counters
-        if self.phy_type == Vsc85x2Type::Vsc8552 {
-            return Ok((Counter::Unavailable, Counter::Unavailable));
+    fn select_media_counters(&mut self) -> Result<(), VscError> {
+        // Configure the PHY to read fiber media SerDes counters
+        if self.phy_type == Vsc85x2Type::Vsc8562 {
+            self.phy.modify(
+                phy::EXTENDED_3::MEDIA_SERDES_TX_CRC_ERROR_COUNTER(),
+                |r| r.set_tx_select(0),
+            )?;
+            self.phy.modify(
+                phy::EXTENDED_3::MEDIA_MAC_SERDES_RX_CRC_CRC_ERR_COUNTER(),
+                |r| r.0 &= !(0b11 << 14),
```

Comment:
I assume there's no convenient/correct helper method for this like there is for `set_tx_select()` above?

Reply:
Unfortunately, no – these registers are all automatically extracted from the SDK, which only defines _some_ of the bits.